src/utils/categorizeCatalogItems.js

```diff
@@ -0,0 +1,233 @@
+import * as _ from 'lodash-es';
```

Comment:
why do we need this external lodash library ?

most of the code is available on ES6. (like forEach, map, etc)